metrics/ci_integrations/test/client/github_actions/models/workflow_run_test.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:ci_integration/client/github_actions/mappers/run_conclusion_mapper.dart';
+import 'package:ci_integration/client/github_actions/mappers/run_status_mapper.dart';
+import 'package:ci_integration/client/github_actions/models/run_conclusion.dart';
+import 'package:ci_integration/client/github_actions/models/run_status.dart';
+import 'package:ci_integration/client/github_actions/models/workflow_run.dart';
+import 'package:test/test.dart';
+
+// https://github.com/software-platform/monorepo/issues/140
+// ignore_for_file: prefer_const_constructors
+
+void main() {
+  group("WorkflowRun", () {
+    final statusMapper = RunStatusMapper();
+    final conclusionMapper = RunConclusionMapper();
```

Comment:
Should we avoid using such a dependency for test data? Seems like we try hardcoding values to ensure that everything is good in methods implementation.